helm/nessie/values.yaml

```diff
@@ -0,0 +1,79 @@
+replicaCount: 1
+
+image:
+  repository: projectnessie/nessie
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+# which type of version store to use: JGIT, INMEMORY, DYNAMO. JGIT is best for local testing, DYNAMO preferred for production
+versionStoreType: INMEMORY
+
+# this is needed when selecting DYNAMO
+dynamodb:
+  region: us-west-2
+  awsAccessKeyId: test-key
+  awsSecretAccessKey: test-secret
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations:
+  prometheus.io/scrape: "true"
+  prometheus.io/path: /q/metrics
+  prometheus.io/port: "19120"
+  prometheus.io/scheme: http
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
```

Comment:
An example of configuring this with a real ingest would be good for the docs

Reply:
I'll try and add a script that generates some data/commits and update the docs in a separate PR

Reply:
by ingest I meant `nginx` and possibly ssl as opposed to ingesting data w/ nessie